### PR TITLE
Added test for removeFund

### LINKED ISSUE #2064: Test: Create tests for src/resolvers/Mutation/removeFund.ts
We will need unittests done for all methods, classes and functions found in this file.

Any widgets, components, modals referenced in this file must also have unittests done

### PR Acceptance Criteria

- When complete this file must show at least **100%** coverage when merged into the code base. 
- [The current code coverage for the file can be found here](https://app.codecov.io/gh/PalisadoesFoundation/talawa-api?search=&displayType=list)
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide
- Please coordinate **issue assignment** and **PR reviews** with the contributors listed in this issue https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by krishna619**
**What kind of change does this PR introduce?**
Test-Added test for removeFund

**Issue Number:**

Fixes #2064

**Did you add tests for your changes?**

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by krishna619** [comment]
@palisadoes please review.

File: src/resolvers/Mutation/removeFund.ts [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const removeFund: MutationResolvers["removeFund"] = async (
     .select("organizationId")
     .lean();
 
-  const currentOrgId = currentOrg?.organizationId?.toString() || "";
+  const currentOrgId = currentOrg?.organizationId?.toString();
```
Comment (palisadoes) [inline src/resolvers/Mutation/removeFund.ts:70]: Why is this changed and not the test?

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: src/resolvers/Mutation/removeFund.ts (hunk 1 above)
Comment (krishna619) [reply src/resolvers/Mutation/removeFund.ts:70]: This line was missing test coverage, and the unnecessary string concatenation was discovered while looking at the code. 
Although the existing tests passed, so there was no need to change the test file or add any tests.

**Review by palisadoes — APPROVED** [review]
